content/templates/output-formats.md

```diff
@@ -122,9 +122,19 @@ The following is the full list of configuration options for output formats and t
 
 ## Output Formats for Pages
 
-A `Page` in Hugo can be rendered to multiple representations on the file system. By default, all pages will render as `HTML` with some of them also as `RSS` (homepage, sections, etc.).
+A `Page` in Hugo can be rendered to multiple *output formats* on the file
+system.
 
-This can be changed by defining an `outputs` list of output formats in either the `Page` front matter or in the site configuration (either for all sites or per language).
+### Default Output Formats
+By default, **all** pages will render to the `HTML` output format. Every `Page`
+also has a [`Kind`][page kinds]. Except for the `page` `Kind`, by default, all
+other *Kinds* will also render to the `RSS` output format.
+
```

Comment:
I think the language here is a little technical.

I would suggest something in the line of: "By default, all ... HTML ... Also, every list page will, by default, render to the `RSS` output format."  

"list page" may not be perfect, but it is easy for people to understand.

Reply:
But it's clearer that way. People will anyways needs to understand Kinds to be able to configure outputs.

Reply:
And also, doesn't feel right mis-documenting.. RSS, 404, etc kinda are not "list pages".. I think that's what you were inferring too.

Reply:
I have attempted to make this  a bit more clearer in my last force-pushed commit to this PR, while retaining the correctness.